Library/src/CSP/Systems/EventTicketing/EventTicketing.cpp

```diff
@@ -35,6 +35,27 @@ csp::systems::EventTicketingVendor VendorNameToEnum(const csp::common::String& V
 	return Vendor;
 }
 
+csp::systems::TicketStatus TicketStatusToEnum(const chs::TicketStatus& DtoStatus)
+{
+	csp::systems::TicketStatus Status = csp::systems::TicketStatus::Unknown;
+
+	if (DtoStatus.GetValue() == chs::TicketStatus::eTicketStatus::PURCHASED)
+	{
+		Status = csp::systems::TicketStatus::Purchased;
+	}
+	if (DtoStatus.GetValue() == chs::TicketStatus::eTicketStatus::REDEEMED)
+	{
+		Status = csp::systems::TicketStatus::Redeemed;
+	}
+	else
+	{
+		FOUNDATION_LOG_MSG(csp::systems::LogLevel::Warning,
+						   "Encountered an unknown ticket status when parsing a response from services. Defaulting to 'Unknown'");
```

Comment:
Possibly this should be error-level? If this happens it's bad for the user as they'll likely hit a scenario where they've tried to use a ticket they got for a space, but due to a code error, they won't able to enter the space.

Reply:
Agreed!